js/constants/countries.js

```diff
@@ -0,0 +1,1000 @@
+const countries = [
```

Comment:
exactly 😄 

It might look like:
```
const countryCodes = require('path/to/countryCodes')
return [
    'downloadsManager',
    'confirmClearPasswords',
     //more strings here
    'channelBeta'
].concat(countryCodes)
```